gf-webapp-admin/src/component/sideMenu/SideMenu.module.scss

```diff
@@ -9,6 +9,30 @@
   align-items: center;
   gap: 30px;
   padding: 20px;
+  
+  label {
+    display: flex;
+    flex-direction: column;
+    max-width: 300px;
```

Comment:
) I know that we agreed to create specific task to move scss variables into separate file. But this doesn't mean that we should add more "bad" code to our project, right? If we cant handle some problem at the moment doesn't mean we should expand it over whole project)
Don't use bad decision even if it is currently exists in project)
(create specific CSS variables)
but dont't all of the at the same file) we should think how to handle it right way

Reply:
These styles will be encapsulated in <InputComponent />. In this case we dont need create constants.